### PR TITLE
mods to grammar/wording in lbc chap (6)

### PR DESCRIPTION
**Pull request by kkeene44**

### CODE REVIEW

**Comment by kkeene44** [comment]
@davegill @dudhia 
Please check the modifications I made when you get a chance.

File: lbc.tex [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-Several lateral boundary condition options exist for the ARW 
-that are suitable for idealized flows, and a specified lateral boundary
-condition for real-data simulations is available.  These choices are
+The ARW provides several lateral boundary conditions suitable 
+for idealized flow, in addition to a specified lateral boundary
+condition for real-data simulations.  These choices are
 handled via a run-time option in the Fortran namelist file.
 For nesting, all fine domains use the 
 nest time-dependent lateral boundary condition where the outer row and column of the
```
Comment (weiwangncar) [inline lbc.tex]: @davegill For a nest, do we specify a single row and column or two of them by the parent domain data?

File: lbc.tex [hunk 2]
```diff
@@ -118,15 +118,15 @@ \section{Specified Lateral Boundary Conditions}
  \centering
   \includegraphics *[width=4.5in]{figures/lbc_zones.pdf}
   \caption{\label{figure:spec} Specified and relaxation zones for a grid
-with a single specified row and column, and four rows and columns for the
+with a single-specified row and column, plus four rows and columns for the
```
Comment (weiwangncar) [inline lbc.tex]: @davegill  Should we say three here? Or should we add the most inner row and column use model data only?

File: lbc.tex (hunk 1 above)
Comment (davegill) [reply lbc.tex]: @weiwangncar
We use one row / column for the `spec_zone` for both the `nested` and `specified` LBC options.
```
     IF (specified) THEN

       ! Arrays for specified boundary conditions

       DO loop = spec_zone + 1, spec_zone + relax_zone
         fcx(loop) = 0.1 / dt * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         gcx(loop) = 1.0 / dt / 50. * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         spongeweight=exp(-(loop-(spec_zone + 1))*spec_exp)
         fcx(loop) = fcx(loop)*spongeweight
         gcx(loop) = gcx(loop)*spongeweight
       ENDDO

     ELSE IF (nested) THEN

       ! Arrays for specified boundary conditions

       DO loop = spec_zone + 1, spec_zone + relax_zone
         fcx(loop) = 0.1 / dt * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         gcx(loop) = 1.0 / dt / 50. * (spec_zone + relax_zone - loop) / (relax_zone - 1)
       ENDDO

     ENDIF
```

File: lbc.tex (hunk 2 above)
Comment (davegill) [reply lbc.tex]: @weiwangncar 
Now this is more clearly stated.

File: lbc.tex [hunk 3]
```diff
@@ -201,20 +203,25 @@ \section{Specified Lateral Boundary Conditions}
...
+By default, all scalars except for vapor have flow-dependent boundary
+conditions applied in the specified zone. This boundary condition is defined as zero-value on inflow
 and zero-gradient on outflow. Since these boundary conditions require only information from
-the interior of the grid, these variables are not in the specified boundary condition file.
-However, for nested boundaries, microphysical variables and scalars are also available
-from the parent domain, and use same relaxation towards interpolated variables as the
+the interior of the grid, these scalar variables are not required to be in the lateral boundary condition file.
+If sufficient scalar data is provided to the lateral boundary condition file, the same technique of
+relaxation
+towards temporally interpolated variables is employed, identically as with the
```
Comment (kkeene44) [inline lbc.tex]: @davegill 
Make 'temporally interpolated' hyphenated, and then I'm fine with these additions.

Comment (davegill) [reply lbc.tex]: Done

**Pull request merged**